app/Widget/WidgetServiceProvider.php

```diff
@@ -86,7 +86,9 @@ public function register(Container $pimple)
                 new TranslateTerm(
                     $pimple['translator']
                 ),
-                $pimple['translator']
+                $pimple['translator'],
+                $pimple['curatoren_api'],
+                $pimple['curatoren_api_test']
```

Comment:
We only added one argument to the `TwigPreprocessor` constructor, why are we adding two here?